src/main/flight/pid.c

```diff
@@ -292,7 +293,17 @@ void pidUpdateTpaFactor(float throttle)
     } else {
         tpaRate = pidRuntime.tpaLowMultiplier * (pidRuntime.tpaLowBreakpoint - throttle);
     }
+
+    DEBUG_SET(DEBUG_TPA, 0, lrintf(pidRuntime.tpaFactor * 1000));
     pidRuntime.tpaFactor = 1.0f - tpaRate;
```

Comment:
Personally, I'd use local variable - write `pidRuntime` only once.
```suggestion
    float tpaFactor = 1.0f - tpaRate;
```

Reply:
I think in case case of a local variable and if `USE_WING` is **not** defined then the assignment will be done twice. Well unless compiler optimizes it. Was trying to make it explicit that its just one assignment, when `USE_WING` is not defined.

Reply:
gcc is very effective with local optimization. 
with local variable, generated code will be probably identical. But it is guaranteed that pidruntime is never in inconsistent state.

Reply:
ok, done!